alchemy/src/supabase/function.md

```diff
@@ -0,0 +1,220 @@
+# Supabase Function Resource
+
+The `Function` resource manages Supabase Edge Functions, which provide serverless compute capabilities at the edge.
+
+## Usage
+
+```typescript
+import { Function } from "alchemy/supabase";
+
+// Create a new Edge Function
+const func = Function("hello-world", {
+  projectRef: "proj-123",
+  body: `
+    export default function handler(req: Request) {
+      return new Response("Hello World!");
+    }
+  `,
+});
+
+// Create function with import map and JWT verification
+const func = Function("api-handler", {
+  projectRef: "proj-123",
+  body: `
+    import { serve } from "https://deno.land/std@0.168.0/http/server.ts";
+    
+    serve((req) => {
+      return new Response("API Response");
+    });
+  `,
```

Comment:
Can we add support for an entrypoint file path that works like alchemy's cloudflare/worker that uses esbuild to bundle instead of requiring it be a string. 

Maybe call it `main` instead of `entrypoint`.